src/Connections/GenericHook.ts

```diff
@@ -332,7 +332,7 @@ export class GenericHookConnection extends BaseConnection implements IConnection
         ctx.runtime.setInterruptHandler(shouldInterruptAfterDeadline(Date.now() + TRANSFORMATION_TIMEOUT_MS));
         try {
             ctx.setProp(ctx.global, 'HookshotApiVersion', ctx.newString('v2'));
-            const ctxResult = ctx.evalCode(`const data = ${JSON.stringify(data)};\n\n${this.state.transformationFunction}`);
+            const ctxResult = ctx.evalCode(`const data = ${JSON.stringify(data)};\n(() => { ${this.state.transformationFunction} })();`);
```

Comment:
Obviously users can break out of this function, but I don't think that makes much difference.

Reply:
True, it shouldn't make a difference because nothing comes after the arrow function, so nothing would happen after the return.